test/internal/modules/charge-information/controllers/charge-element.js

```diff
@@ -0,0 +1,168 @@
+'use strict';
+
+const { expect } = require('@hapi/code');
+const {
+  experiment,
+  test,
+  beforeEach,
+  afterEach
+} = exports.lab = require('@hapi/lab').script();
+const sinon = require('sinon');
+const moment = require('moment');
+const uuid = require('uuid/v4');
+const sandbox = sinon.createSandbox();
+const formHelpers = require('../../../../../src/shared/lib/forms');
+
+const controller = require('../../../../../src/internal/modules/charge-information/controllers/charge-element');
+
+const routingConfig = {
+  purpose: { pageTitle: 'Select a purpose use', nextStep: 'description' },
+  description: { pageTitle: 'Add element description', nextStep: 'abstraction', back: 'purpose' },
+  abstraction: { pageTitle: 'Set abstraction period', nextStep: 'quantities', back: 'description' },
+  quantities: { pageTitle: 'Add licence quantities', nextStep: 'time', back: 'abstraction' },
+  time: { pageTitle: 'Set time limit?', nextStep: 'source', back: 'quantities' },
+  source: { pageTitle: 'Select source', nextStep: 'season', back: 'time' },
+  season: { pageTitle: 'Select season', nextStep: 'loss', back: 'source' },
+  loss: { pageTitle: 'Select loss category', nextStep: 'loss', back: 'season' }
+};
```

Comment:
If you require this from the controller, or put it in it's own file, it will make it easier when making changes to this flow in future

Reply:
done thanks